packages/react/src/themes/teams-high-contrast/siteVariables.ts

```diff
@@ -28,6 +28,12 @@ export const errorStatusTextColor = black
 export const unknownStatusBackgroundColor = colors.white
 export const unknownStatusTextColor = black
 
+//
+// BORDER STYLES
+//
+export const focusInnerBorderColor = colors.black
```

Comment:
Hmm, I thought the focused border looks exactly the same in light and dark contrast. BTW why we don't have mechanism for high contrast theme? I think it is suppose to be accessibleCyan in that case. @codepretty ?

Reply:
High contrast has different requirements. we're supposed to be mapping to what Windows does by default which isn't always clear. Depending on the context, some things will have a background. i'm not sure how to standardize high contrast yet.

Reply:
I think the current design is:
- Regular theme: `focusInnerBorderColor: white`, `focusOuterBorderColor: black`
- Dark theme: `focusInnerBorderColor: black`, `focusOuterBorderColor: white` (inverted basically)
- HC theme: same as Dark

@codepretty to confirm

Reply:
the current design is:

Regular theme: focusInnerBorderColor: white, focusOuterBorderColor: black
Dark theme: focusInnerBorderColor: white, focusOuterBorderColor: black
HC theme: focusInnerBorderColor: cyan, focusOuterBorderColor: black

Reply:
Made the changes. Here are screenshots for `Button`:

#### Regular Theme
![Screenshot 2019-05-06 at 11 40 42](https://user-images.githubusercontent.com/5442794/57217866-17973680-6ff4-11e9-89cc-4f491d6741a1.png)

#### Dark Theme
![Screenshot 2019-05-06 at 11 41 01](https://user-images.githubusercontent.com/5442794/57217869-19f99080-6ff4-11e9-99f8-cbb5a1216936.png)

#### High Contrast Theme
![Screenshot 2019-05-06 at 11 41 23](https://user-images.githubusercontent.com/5442794/57217872-1bc35400-6ff4-11e9-9eec-0f028dc9d4ad.png)

However, as you can see, items seem smaller in Dark and HC theme because the white border is more visible:
#### in Regular Theme:
we have **light background -> dark border -> light border -> content**

#### in Dark and HC Theme:
we have **dark background -> dark border\* -> light border -> content**
\* - dark background next to dark border => component seems smaller 
![Screen Recording 2019-05-06 at 11 52 54](https://user-images.githubusercontent.com/5442794/57218452-a062a200-6ff5-11e9-8396-c1ea118148bc.gif)


I believe it'd be better if we swapped colors for Dark and HC theme and have it like:
1. Regular theme: `focusInnerBorderColor: white`, `focusOuterBorderColor: black`
2. Dark theme: `focusInnerBorderColor: black`, `focusOuterBorderColor: white`
3. HC theme: `focusInnerBorderColor: black`, `focusOuterBorderColor: cyan`

@codepretty what do you think?

Reply:
Had a conversation with @codepretty and we decided to invert colors